report_thesis/src/sections/introduction.tex

```diff
@@ -5,30 +5,42 @@ \section{Introduction}\label{sec:introduction}
+Furthermore, the high cost of data collection often results in small datasets, exacerbating the difficulty of building accurate and robust models.
 
-Various machine learning models have been used to predict the composition of major oxides in the sample, including \glspl{cnn}~\cite{yang_laser-induced_2022, yangConvolutionalNeuralNetwork2022}, \gls{svr}~\cite{rezaei_dimensionality_reduction}, and hybrid models like \gls{df}-\gls{k-elm}~\cite{song_DF-K-ELM} that incorporate domain knowledge to enhance model interpretability and performance.
-However, the high dimensionality and multicollinearity of the spectral data remains a significant challenge for these models.
+Previous work has aimed to improve the prediction of major oxide compositions from LIBS data.
+Regression techniques and dimensionality reduction with feature selection have been used to enhance the accuracy and interpretability of these models.
+Tailored approaches have also been developed, where different models are selected based on their performance with specific spectral characteristics and distances~cite{rezaei_dimensionality_reduction, andersonPostlandingMajorElement2022}.
+Moreover, models incorporating physical principles have demonstrated improved accuracy by handling residuals that traditional models fail to explain~\cite{song_DF-K-ELM}.
+However, predicting oxide compositions remains challenging due to the complex, nonlinear nature of LIBS data.
+This underscores the need for continued research into more adaptive and robust machine learning strategies to tackle these issues effectively.
 
-Building upon the baseline established in~\citet{p9_paper}, this thesis aims to explore approaches for tackling the challenges in predicting major oxide compositions from \gls{libs} data. We develop machine learning models that seek to enhance the accuracy and robustness of these predictions.
+This thesis aims to improve the prediction of major oxide compositions from LIBS data by building on the foundation established in~\citet{p9_paper} in order to enhance model accuracy and robustness.
 We define accuracy as the ability of a model to predict the composition of major oxides in Martian geological samples, while robustness refers to the stability of these predictions across different samples and oxides.
 
-We investigate various techniques to handle the high dimensionality, non-linearity, and small dataset size inherent in this problem, and evaluate the performance of these models using appropriate metrics. 
-Through extensive experiments on \gls{libs} data, we demonstrate the superior performance of our approach compared to existing methods in terms of both prediction accuracy and computational efficiency.
+To achieve these objectives, this research systematically explores a range of promising machine learning models and preprocessing techniques, identified through an extensive literature review and guided by a curiosity to explore unconventional approaches.
```

Comment:
Wonder if it would make sense to put the "we build upon (p9)" here? As part of how we achieve the objectives.

Reply:
Rephrased